### PR TITLE
Updated error message.

### PR DESCRIPTION
**Pull request by lu-ohai**

### CODE REVIEW

File: ads/aqua/__init__.py [hunk 1]
```diff
@@ -26,3 +26,8 @@
         logger.error(
```
Comment (VipulMascarenhas) [inline ads/aqua/__init__.py:26]: logging this inside Exception wouldn't cover the case if `fetch_service_compartment` returns None. Can we do something like:
```
except Exception as e:
    pass
if not ODSC_MODEL_COMPARTMENT_OCID:
     logger.error(
        f"ODSC_MODEL_COMPARTMENT_OCID environment variable is not set for AQUA."
    )
    if NB_SESSION_OCID:
        logger.error(
            "AQUA is not available for this notebook session."
        )
    exit()
```

Comment (lu-ohai) [reply ads/aqua/__init__.py:26]: Oh I was updating it the same as you suggested in the ticket. Could you be more specific or we can have a zoom call on this?

**Comment by mingkang111** [comment]
Can you please rebase the `feature/aquav1.0.1` ? The `feature/aquav1.0.1` just sync with `main`.

File: ads/aqua/__init__.py (hunk 1 above)
Comment (lu-ohai) [reply ads/aqua/__init__.py:33]: Just updated. Could you take a look now?

**Review by VipulMascarenhas — APPROVED** [review]
changes look good, thank you!

**Pull request merged**